src/main/java/com/griddynamics/controllers/ProductController.java

```diff
@@ -71,23 +83,29 @@ public ProductController(ProductService productService) {
         ProductDTO updatedProduct;
 
         try {
+            log.info("Trying to update product");
             updatedProduct = productService.update(product);
         } catch (ServiceException e) {
+            log.error("Can't update product");
             return new ResponseEntity<>(HttpStatus.BAD_REQUEST);
         }
 
+        log.info("Product updated successfully");
         return new ResponseEntity<>(updatedProduct, HttpStatus.OK);
     }
 
     @DeleteMapping(value = "/product/{id}", produces = MediaType.APPLICATION_JSON_VALUE)
     public ResponseEntity<ProductDTO> deleteProduct(@PathVariable Integer id) {
 
         try {
+            log.info("Trying to delete product with id " + id);
             productService.deleteById(id);
         } catch (ServiceException e) {
+            log.error("Product with id " + id + " not found");
```

Comment:
Please take a look on exceptionHandlers in spring
this will help to avoid code duplicate and will provide more readable code